scripts/gan/stylegan/demo_stylegan.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import math
+import os
+import numpy as np
+from tqdm import tqdm
+from PIL import Image
+
+import mxnet as mx
+import mxnet.ndarray as nd
+
+from model import StyledGenerator
+# pylint: disable-all
+
+def get_mean_style(generator, device):
+    mean_style = None
+
+    for i in range(10):
+        style = generator.mean_style(nd.random.randn(1024, 512, ctx=device))
+
+        if mean_style is None:
+            mean_style = style
+
+        else:
+            mean_style += style
+
+    mean_style /= 10
+    return mean_style
+
+
+def sample(generator, step, mean_style, n_sample, device):
+
+    image = generator(
+        nd.random.randn(n_sample, 512, ctx=device), step, 1, mean_style, 0.7,
+    )
+
+    return image
+
+
+def normalize_image(img, dmin, dmax):
+    result = img.copy()
+    result = nd.clip(img, dmin, dmax)
+    result = (result - dmin)/(dmax - dmin + 1e-5)
+
+    return result
+
+
+def save_image(data, file, normalize=True, img_range=None):
+
+    if img_range is None:
+        img_range = [min(data), max(data)]
+
+    norm_img = normalize_image(data, img_range[0], img_range[1])
+    img = nd.clip(norm_img * 255 + 0.5, 0, 255).asnumpy().astype(np.uint8) 
+    img = Image.fromarray(np.transpose(img, (1, 2, 0)))
+    img.save(file)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
```

Comment:
Can you write a README for this folder, like cycleGAN folder did? For example, how to prepare the training data, how to train and how to evaluate the model, etc. Thank you.

Reply:
> Can you write a README for this folder, like cycleGAN folder did? For example, how to prepare the training data, how to train and how to evaluate the model, etc. Thank you.

Sure, no problem.

Reply:
I've added README. Please check.